src/plugin.js

```diff
@@ -12,9 +12,10 @@ var plugin = function(mail, done) {
     var attachments = {};
 
     html = replaceBase64Images(html, function(mimeType, base64) {
+
       if (attachments[base64]) return attachments[base64].cid;
 
-      var randomCid = crypto.randomBytes(8).toString('hex');
+      var randomCid = crypto.randomBytes(8).toString('hex') + '@nodemailer.com';
```

Comment:
It's interesting that Gmail's own Content-IDs don't have an @: `Content-ID: <ii_iu2vlgi02_157aa636a6891103>`. Just curious, what mail client didn't show the image for you when you tested Content-IDs without @?